kvac/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.0.5"
+__version__ = "1.0.0"
```

Comment:
```suggestion
__version__ = "0.1.0"
```

1.0.0 suggests that the library is mature.